lib/conjur/fpm/Dockerfile

```diff
@@ -1,14 +1,14 @@
 # Build from the same version of ubuntu as phusion/baseimage
-FROM ubuntu:14.04
+FROM ubuntu:18.04
 
 RUN apt-get update -y && \
     apt-get install -y software-properties-common && \
     apt-add-repository -y ppa:brightbox/ruby-ng && \
     apt-get update -y && \
-    apt-get install -y build-essential git libpq5 libpq-dev ruby2.2 ruby2.2-dev libffi-dev
+    apt-get install -y build-essential git libpq5 libpq-dev ruby2.3 ruby2.3-dev libffi-dev
 
 
-# Configure bundler and gem the way the ruby:2.2 Dockerfile does, as of 
+# Configure bundler and gem the way the ruby:2.2 Dockerfile does, as of
```

Comment:
shouldn't this be changed to ruby 2.3 as well?

Reply:
Good catch, will update.

Reply:
Actually changed my mind, its not saying that ruby2.3 is installed, the comment is talking about setting up bundler in the same way as another dockerfile, so I now think it should stay as is.

Reply:
I still think that since you've updated to ruby 2.3 you should take the 2.3 docker file example.
The bundler version is the same in any case

Reply:
As far as I can tell, the [ruby:2.3 Dockerfile](https://github.com/docker-library/ruby/blob/802421922ef50cfa05c89a3c619992acf4329986/2.3/stretch/Dockerfile) doesn't contain `BUNDLER_VERSION`, so updating the comment to point to that would probably cause confusion.

Reply:
look at this file.
Anyway - your choice. I don't feel it's a show stopper

https://github.com/docker-library/ruby/blob/c88f3a67da720bfa9fb1717960d90fd5db11c757/2.3/Dockerfile

Reply:
Hah thats the commit the comment already points to. I've updated the comment to specify 2.3 and to link to the exact file/line so it's useful to future readers.